mvp-spec/signatures.md

```diff
@@ -0,0 +1,62 @@
+# Signatures
+
+A quick introduction on how data (id, preferences) and messages (operator API, RTB) are signed and how these signatures can be verified.
+
+PAF Data format is designed to let the users audit how their preferences got
+to their current state. It relies on the signatures of data and communication, to enforce security.
+
+All "signers" have an Elliptic Curve (EC) **private** and **public** key using the `secp256r1` curve.
+
+## Signing an object
+
+A "signer" needs to calculate the signature to associate with an object (cookie or message).
+
+1. the signer computes the _signature input_ for the object to sign
+   1. usually, different properties from the object are "joined together" with the special separator character `\u2063`
+   2. but **each type of object has its own rule to calculate the signature input**. Refer to documentation these rules.
+
+Example:
+```
+transmission_result.source.domain + '\u2063' + 
+transmission_result.source.timestamp + '\u2063' + 
+seed.source.signature + '\u2063' + 
+source.domain + '\u2063' + 
+source.timestamp + '\u2063' + 
+transmission_response.receiver + '\u2063' + 
+transmission_response.status + '\u2063' +
+transmission_response.details
+```
```

Comment:
Our context with the pending pull requests about DPS and Ad Server is annoying because we have a partial for this one in it. Maybe we can extract some partials to a dedicated branch to merge it quickly so that you can reuse it. WDYT ?

Reply:
Yes, we could do this of course!
Although this is just an example, so we might as well put something random here to not depend on a partial.